R/tbl_ard_hierarchical.R

```diff
@@ -0,0 +1,115 @@
+#' ARD Hierarchical Table
+#'
+#' @description `r lifecycle::badge('experimental')`\cr
+#' *This is an preview of this function. There will be changes in the coming releases, and changes will not undergo a formal deprecation cycle.*
+#'
+#' Calculates *rates* of events (e.g. adverse events).
```

Comment:
May want to change this text since it depends on this input (i.e. `ard_stack_hierarchical() vs. `ard_stack_hierarchical_count()`).

Reply:
ahh, thank you!! updated to `"Constructs tables from nested or hierarchical data  structures (e.g. adverse events)."`